projects/core/src/main/resources/data/computercraft/lua/rom/apis/textutils.lua

```diff
@@ -940,6 +940,32 @@ function urlEncode(str)
     return str
 end
 
+--- Splits a string in a table.
+--
+-- @tparam string string_to_split The string to split.
+-- @tparam string pattern At what pattern to split the string.
+-- @treturn table A table containing the splitted strings.
+-- @usage args = textutils.splitString("arg1 arg2", " ")
+-- @since edit this if it goes live (I hope it does)
+local function split(string_to_split, pattern)
```

Comment:
```suggestion
function split(string_to_split, pattern)
```

`textutils` uses the old style of module building, so this function should not have the `local` identifier.
Edit: this is also the reason the linting is failing.

Reply:
Ohh thanks I removed it first but forgot to remove it again after modifying the var names 😅